src/game_main.cpp

```diff
@@ -209,6 +211,43 @@ std::unique_ptr<GameMode> createInitialGameMode(
 }
 
 
+void showErrorMessage(SDL_Window* pWindow, const std::string& error) {
```

Comment:
is there a better place to move this ? like utility namespace ?

Reply:
Yep, let's move it once we start using it in other places :+1: